.github/workflows/pull-request.yml

```diff
@@ -247,161 +260,26 @@ jobs:
 
       - name: resolve dependency
         run: npm install -g @graphql-inspector/cli
-
+      
       - name: Clone API Repository
         run: |
           # Retrieve the complete branch name directly from the GitHub context
           FULL_BRANCH_NAME=${{ github.base_ref }}
           echo "FULL_Branch_NAME: $FULL_BRANCH_NAME"
-
+          
           # Clone the specified repository using the extracted branch name
-          git clone --branch $FULL_BRANCH_NAME https://github.com/PalisadoesFoundation/talawa-api && ls -a
+          git clone --branch $FULL_BRANCH_NAME https://github.com/PalisadoesFoundation/talawa-api && ls -a 
 
       - name: Validate Documents
         run: graphql-inspector validate './src/GraphQl/**/*.ts' './talawa-api/schema.graphql'
 
-  Start-App-Without-Docker:
-    name: Check if Talawa Admin app starts (No Docker)
-    runs-on: ubuntu-latest
```

Comment:
@prayanshchh 

This is one of the most serious reversions of the code. We recently merged a PR that would test whether the app would start. Here you reverted the entire test.